modules/core/04-channel/keeper/upgrade.go

```diff
@@ -110,6 +110,29 @@ func (k Keeper) WriteUpgradeTryChannel(
 	emitChannelUpgradeTryEvent(ctx, portID, channelID, channel, proposedUpgrade)
 }
 
+// WriteUpgradeCancelChannel writes a channel which has canceled the upgrade process. Auxiliary upgrade state is
+// also deleted.
+func (k Keeper) WriteUpgradeCancelChannel(ctx sdk.Context, portID, channelID string) error {
+	defer telemetry.IncrCounter(1, "ibc", "channel", "upgrade-cancel")
+
+	channel, found := k.GetChannel(ctx, portID, channelID)
+	if !found {
+		panic(fmt.Sprintf("could not find existing channel when cancelling channel upgrade, channelID: %s, portID: %s", channelID, portID))
```

Comment:
Is it really necessary to add `when cancelling channel upgrade` to the error message? Wouldn't there be any other information in the log (like a stack trace) to figure that out?

Reply:
yeah you're right it's not really required